BREAKING.md

```diff
@@ -138,9 +138,15 @@ Please move all usage to the new `minIdleTime` and `maxIdleTime` properties in `
 # 2.0.0-internal.1.3.0
 
 ## 2.0.0-internal.1.3.0 Upcoming changes
```

Comment:
@tylerbutler is this the right section at this point in time? I would think main bumped to internal-2.1.0